app/pages/dashboard.vue

```diff
@@ -1,246 +1,276 @@
-        <h1 class="w-full xl:w-6/12 xl:px-5 lg:px-10 lg:w-full md:w-9/12 m-auto uppercase font-bold mt-1">Actions</h1>
-        <ul class="w-full xl:w-6/12 xl:px-5 lg:px-10 lg:w-full md:w-9/12 m-auto mt-2 text-center">
-            <li class="py-2 bg-emerald-700 text-white rounded hover:bg-emerald-600 transition-colors w-40 font-semibold uppercase"
-                style="font-size: smaller;">
-                <NuxtLink to="/forms/create">Create A Form</NuxtLink>
-            </li>
-        </ul>
-    </div>
-</template>
 <script setup lang="ts">
+import { DollarSignIcon, FileTextIcon, GitGraphIcon } from 'lucide-vue-next';
+import type { Component } from 'vue';
+
+
 definePageMeta({
-    middleware: ['auth']
-})
+  middleware: ["auth"],
+});
 
-const formCount = ref(0);
-const moneyCount = ref(0);
-const reachCount = ref(0);
+type CardData = {
+  url: string;
+  title: string;
+  count: number;
+  description: string; // Can Have Html format
+  icon: Component
+};
 
-const stats = await useFetch<APIResponse<{ forms: number, responses: number, earnings: number }>>("/api/v1/forms/me/stats", {
-    headers: {
-        Authorization: `Bearer ${getAuthToken()}`,
-    },
-    onResponseError({ response }) {
-        console.error(response)
-    },
-    onRequestError({ error }) {
-        console.error(error)
-    }
-}).then(({ data }) => data.value?.body).catch(err => { return {} as any; })
-formCount.value = stats?.forms || 0;
-moneyCount.value = stats?.earnings || 0;
-reachCount.value = stats?.responses || 0;
-const recents = await useFetch<APIResponse<{ forms: Drizzle.Form.select[] }>>(`/api/v1/forms/me/recents`, {
+// Vars
+
+const cards = ref<CardData[]>([]);
+
+// Composables
+const stats = await useFetch<
+  APIResponse<{ forms: number; responses: number; earnings: number }>
+>("/api/v1/forms/me/stats", {
+  headers: {
+    Authorization: `Bearer ${getAuthToken()}`,
+  },
+  onResponseError({ response }) {
+    console.error(response);
+  },
+  onRequestError({ error }) {
+    console.error(error);
+  },
+})
+  .then(({ data }) => data.value?.body)
+  .catch((err) => {
+    return {} as any;
+  });
+
+cards.value = [
+  {
+    title: "FORMS",
+    count: stats?.forms || 0,
+    url: "/forms",
+    description: `<span>You have created</span><span class="font-medium text-gray-900 mx-1">${stats?.forms || 0}</span>
+              <span>forms so far</span>`,
+    icon: FileTextIcon,
+  },
+  {
+    title: "MONEY",
+    count: stats?.earnings || 0,
+    url: "/finance",
+    description: `<span>You have earned</span>
+<span class="font-medium text-gray-900 mx-1">${stats?.earnings || 0} KES</span>
+              <span>so far</span>`,
+    icon: DollarSignIcon
+  },
+  {
+    title: "REACH",
+    count: stats?.responses || 0,
+    url: "/finance",
+    description: `<span>You have reached</span>
+<span class="font-medium text-gray-900 mx-1">${stats?.responses || 0}</span>
+              <span>people so far</span>`,
+    icon: GitGraphIcon
+  },
+];
```

Comment:
I avoid writing HTML in TS and leave it in the Templates. Transform this into a component instead.

```
defineComponent({...})
```

Check out the docs [here](https://vuejs.org/api/general.html#definecomponent).

Reply:
I Dont think i can define a component within the script setup